src/components/CallView/shared/LocalMediaControls.vue

```diff
@@ -396,6 +391,10 @@ export default {
 	},
 
 	watch: {
+		currentVolumeProportion: function() {
+			this.updateVolumeMeter()
```

Comment:
It could be good to add a small comment explaining why it needs to be done with a watcher instead of with the style.

Reply:
> Tested and works.
> 
> However, personally I would not add the transition. For me it causes the volume indicator to feel "laggy" and less dynamic and responsive to the sound (I guess that if a new height is set while the height was transitioning the old target is ignored and the new value is used as the target of the transition, although I do not really know).

Here it depends what the purpose of the meter is. If this was audio mixing software it would have to be super responsive. I think the purpose of the meter is anyway only to find out whether audio is working.

Did you try reducing the transition delay ? In any case I'm fine removing it completely.

Reply:
I think I'll remove it.

One idea for later to make it look nicer is to reduce the fall speed. So even if we measure zero, the meter would drop more slowly than it raises. But maybe at that point we could also switch to a library. Anyway, something for later.